package day40_exception;

public class UsingThrows {
    public static void main(String[] args) {

    }
}
